### PR TITLE
feat(`NcRichContenteditable`) - add different output appearance in examples 

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Raise awareness how the component handles and transfers strings
* May help to identify some issues before they reach release
* Cover frequent usecases:
  * render as a simple message 
  * send to server with `JSON.stringify()`
  * interact with `<NcRichText />` (with markdown enabled)

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

### 🚧 Tasks

- [ ] Wording suggestions?
- [ ] Another outputs?
- [ ] Visual check

### 🏁 Checklist

- [x] ⛑️ Tests are included or are not applicable
- [x] 📘 Component documentation has been extended, updated or is not applicable

### CODE REVIEW

File: src/components/NcRichContenteditable/NcRichContenteditable.vue [hunk 1]
```diff
@@ -47,16 +47,26 @@ This component displays contenteditable div with automated `@` [at] autocompleti
 			:user-data="userData"
 			placeholder="Try mentioning user @Test01 or inserting emoji :smile"
 			@submit="onSubmit" />
-		<br>
-		<br>
+
+		<h5>Output - default</h5>
+		{{ message }}
```
Comment (ShGKme) [inline src/components/NcRichContenteditable/NcRichContenteditable.vue:52]: I'd add `white-space: pre-wrap` here to display new lines.

**Review by ShGKme — APPROVED** [review]
Should we also add more examples or `NcRichContenteditable` input to the `NcRichText` documentation?

File: src/components/NcRichContenteditable/NcRichContenteditable.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/NcRichContenteditable/NcRichContenteditable.vue:52]: It will be nice, but white spaces are collapsed [by default](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space), we have `pre-wrap` only for NcRichText
I wouldn't confuse other developers on that place (or we could show both options)

Comment (Antreesy) [reply src/components/NcRichContenteditable/NcRichContenteditable.vue:52]: pre-wrap looks weird here, I'd like to not add it for docs

image

File: src/components/NcRichContenteditable/NcRichContenteditable.vue [hunk 2]
```diff
@@ -47,16 +47,28 @@ This component displays contenteditable div with automated `@` [at] autocompleti
...
-		<br>
-		<br>
+
+		<h5>Output - raw</h5>
 		{{ JSON.stringify(message) }}
+
+		<h5>Output - preformatted</h5>
+		<p class="pre-line">
+			{{ message }}
+		</p>
```
Comment (ShGKme) [inline src/components/NcRichContenteditable/NcRichContenteditable.vue:57]: To remove two addition blank lines without `-20px` margin
```suggestion
		<p class="pre-line">{{ message }}</p>
```

File: src/components/NcRichContenteditable/NcRichContenteditable.vue (hunk 1 above)
Comment (ShGKme) [reply src/components/NcRichContenteditable/NcRichContenteditable.vue]: > Do you mean docs block in NcRichText file?

Yep, to have more Markdown examples in the docs like we have in tests and bugs. To see all the "unobvious" edge cases.

**Review by Pytal — APPROVED** [review]
